MSMAll/MSMAllPipeline.sh

```diff
@@ -109,61 +121,58 @@ get_options()
 				;;
 			--path=*)
 				p_StudyFolder=${argument#*=}
-				index=$(( index + 1 ))
 				;;
 			--study-folder=*)
 				p_StudyFolder=${argument#*=}
-				index=$(( index + 1 ))
 				;;
 			--subject=*)
 				p_Subject=${argument#*=}
-				index=$(( index + 1 ))
 				;;
 			--fmri-names-list=*)
 				p_fMRINames=${argument#*=}
-				index=$(( index + 1 ))
 				;;
+			--multirun-fix-names=*)
+			    p_mrfixNames=${argument#*=}
+			    ;;
+			--multirun-fix-concat-name=*)
+			    p_mrfixConcatName=${argument#*=}
+			    ;;
+		    --multirun-fix-names-to-use=*)
+		        p_mrfixNamesToUse=${argument#*=}
+		        ;;
 			--output-fmri-name=*)
 				p_OutputfMRIName=${argument#*=}
-				index=$(( index + 1 ))
 				;;
 			--high-pass=*)
 				p_HighPass=${argument#*=}
-				index=$(( index + 1 ))
 				;;
 			--fmri-proc-string=*)
 				p_fMRIProcSTRING=${argument#*=}
-				index=$(( index + 1 ))
 				;;
 			--msm-all-templates=*)
 				p_MSMAllTemplates=${argument#*=}
-				index=$(( index + 1 ))
 				;;
 			--output-registration-name=*)
 				p_OutputRegName=${argument#*=}
-				index=$(( index + 1 ))
 				;;
 			--high-res-mesh=*)
 				p_HighResMesh=${argument#*=}
-				index=$(( index + 1 ))
 				;;
 			--low-res-mesh=*)
 				p_LowResMesh=${argument#*=}
-				index=$(( index + 1 ))
 				;;
 			--input-registration-name=*)
 				p_InputRegName=${argument#*=}
-				index=$(( index + 1 ))
 				;;
 			--matlab-run-mode=*)
 				p_MatlabRunMode=${argument#*=}
-				index=$(( index + 1 ))
 				;;
 			*)
 				usage
 				log_Err_Abort "unrecognized option: ${argument}"
 				;;
 		esac
+#		index=$(( index + 1 ))
```

Comment:
I'm probably missing something, but isn't `index` incremented as part of the `for` loop construct?  Do we really want to increment it again here?

Reply:
It is commented out.  I can delete it.